node_modules/.package-lock.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
This file is not really needed to be added. Please add the `node_modules` to the `.gitignore` file.